src/SetStateInterface.php

```
@@ -0,0 +1,18 @@
+
+namespace Ray\Aop;
+
+use Ray\Aop\MethodInterceptor as MethodBindings;
+
+/** @psalm-import-type MethodBindings from Types */
+interface SetStateInterface extends WeavedInterface // @phpstan-ignore-line
+{
+    /**
+     * @param MethodBindings $bindings
+     *
+     * @SuppressWarnings(PHPMD.CamelCaseMethodName)
```

PHPStan (2.0.3)

PHPDoc tag @SuppressWarnings has invalid value ((PHPMD.CamelCaseMethodName)): Unexpected token ".CamelCaseMethodName)", expected ')' at offset 80 on line 4


(phpDoc.parseError)